PacketDotNet/DrdaDDMPacket.cs

```diff
@@ -198,18 +206,20 @@ public override string ToString(StringOutputType outputFormat)
                     CorrelId,
                     Length2,
                     CodePoint);
+                buffer.Append(" Paramters:{");
                 foreach(var paramter in Parameters)
                 {
-                    buffer.AppendFormat("{0}[DrdaDDMParameter: Length={2}, CodePoint={3}, Data=\" {4} \"]{1}",
+                    buffer.AppendFormat("{0}[DrdaDDMParameter: Length={2}, CodePoint={3}, Data='{4}']{1}",
                     color,
                     colorEscape,
                     paramter.Length,
                     paramter.DrdaCodepoint,
                     paramter.Data);
                 }
+                buffer.Append("}");
             }
             // append the base class output
-            buffer.Append(base.ToString(outputFormat));
+            //buffer.Append(base.ToString(outputFormat));
```

Comment:
I prefer that we don't include commented out code. If you'd like to keep it you should uncomment it, if it can go you can just delete it.